pack/mts_encounter.json

```
@@ -992,9 +1031,12 @@
 		"quantity": 2,
 		"set_code": "thanos",
 		"set_position": 13,
+		"text": "Incite 1. \n <b> When Revealed: </b> Give Thanos a tough status card. If you cannot, place 2 threat on the main scheme.",
+		"boost_text": "[star] <b>Boost</b> Give Thanos a tough status card.",
```

This should be defined alphabetically, before `code`

```suggestion
		"boost_text": "Give Thanos a tough status card.",
```